Fundamentals/first_letter.py

```diff
@@ -0,0 +1,15 @@
+# Copyright (c) 2016 Ami . All rights reserved.
```

Comment:
Right, good. Just don't forget to use the official version when you start working on production code :)